keyring/backends/macOS/__init__.py

```diff
@@ -1,5 +1,6 @@
 import platform
 import os
+from os.path import exists
```

Comment:
I'd avoid this unnecessary import. `os.path.exists` is already available in the namespace.